src/Sylius/Bundle/AdminBundle/Resources/config/grids/shipping_method.yml

```diff
@@ -7,7 +7,7 @@ sylius_grid:
                     class: "%sylius.model.shipping_method.class%"
                     repository:
                         method: createListQueryBuilder
-                        arguments: ["%locale%"]
+                        arguments: ["expr:service('sylius.context.locale').getLocaleCode()"]
```

Comment:
I changed it because grid filter didn't work properly, Translations worked properly, but grid filter sorted it using default locale, After changed locale for example : "pl_PL", them sort filter ordered by default locale

Reply:
Probably it will fix #11422